EventFilter/CTPPSRawToDigi/python/ctppsRawToDigi_cff.py

```diff
@@ -18,13 +18,13 @@
     ),
     # during TS2 (2016)
     cms.PSet(
-      validityRange = cms.EventRange("280386:min - 281600:max"),
+      validityRange = cms.EventRange("280385:min - 281600:max"),
       mappingFileNames = cms.vstring(),
       maskFileNames = cms.vstring()
     ),
     # after TS2 (2016)
     cms.PSet(
-      validityRange = cms.EventRange("281601:min - 999999999:max"),
+      validityRange = cms.EventRange("281600:min - 999999999:max"),
```

Comment:
Hi @slava77 
It seems that the bug lies further away than this, as the first event in `/store/data/Run2016H/ZeroBias/RAW/v1/000/283/820/00000/462D2A5B-B19A-E611-B100-02163E01382E.root'` (as discussed offline) is `283820:0:0`, while `283820:min` seems to correspond to `283820:0:1`...